packages/launcher/lib/browsers.ts

```diff
@@ -166,7 +166,7 @@ export function launch (
   debuggingPort: number,
   args: string[] = [],
   defaultBrowserEnv = {},
-): LaunchedBrowser {
```

Comment:
Return value is implicit

<img width="656" alt="image" src="https://user-images.githubusercontent.com/19196536/188760950-5e7dabef-4bf0-4e50-a54d-c238002d1902.png">